pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,248 @@
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. During this whole setup, we’ll be working within this repository.\n",
+    "\n",
+    "For successful completion of this guide you need to understand what the aim is. Please have prepared following data:\n",
+    "- Name of the onboarded team.\n",
+    "- Desired namespace name. Please use your team name as a prefix. This will make it easier for you to onboard to ArgoCD later on.\n",
+    "- List of users you’d like to add to your team.\n",
+    "- An optional team GPG key, in case you would like to modify the encrypted list of users of your team later on.\n",
+    "\n",
+    "### Recipe\n",
+    "If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).\n",
+    "In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.\n",
+    "#### Adding namespaces\n",
+    "For easier onboard to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$OWNER_TEAM-example-project`.\n",
+    "#####  Base resources\n",
+    "\n",
+    "Please run:\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!opfcli create-project $NAMESPACE_NAME $OWNER_TEAM -d $OPTIONAL_PROJECT_DESCRIPTION"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "This command will create:\n",
+    "\n",
+    "- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`\n",
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.\n",
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Open the `kustomization.yaml` file in this folder and add `../../base/core/namespaces/$NAMESPACE` to the `resources` field. We aim to keep this field alphabetically sorted for better human readability:\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "...\n",
+    "- ../../base/core/namespaces/$NAMESPACE\n",
+    "...\n",
+    "```\n",
+    "##### Authenticate via OpenShift\n",
```

Comment:
Can you please either remove this section or move it to the bottom? It's not needed here at this stage, just distraction from the tutorial.

Reply:
Resolved in latest commit.